backend/api/models/const.py

```diff
@@ -0,0 +1,23 @@
+class DocumentDisplayStatusConst:
```

Comment:
Append some discussion about the implementation of the namespace https://github.com/tai271828/disfactory-backend/commit/2f59e5f5d1558341704b2d0bbad230f7bcba30b0#commitcomment-61289751